pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -319,422 +320,469 @@ async def run(self):
         remote_url = remote_url if remote_url[-1] == "/" else f"{remote_url}/"
         optimize_sync = self.optimize
 
+        task = asyncio.ensure_future(self._fetch_remote_urls(remote_url))
```

Comment:
I think ```asyncio.create_task()``` is preferred now

https://docs.python.org/3/library/asyncio-future.html#asyncio.ensure_future

Reply:
create_task is preferred, but only aveiable since python 3.7